core/scripts/local/DepositCollateral.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * @notice Deposits collateral into existing token position with --collateral of collateral.
+ *
+ * Example: `$(npm bin)/truffle exec ./scripts/local/DepositCollateral.js --network test --collateral 25 --emp 0x6E2F1B57AF5C6237B7512b4DdC1FFDE2Fb7F90B9 --collateralToken 0xbF7A7169562078c96f0eC1A8aFD6aE50f12e5A99`
```

Comment:
I dont want to nit pick too much but do you really need to provide the `collateralToken` address if you are providing the emp address? it can get this in one async call, rather than the user having to fetch this. other than this script is great!

Reply:
Yeah you're totally right